app/views/teams/show.html.erb

```diff
@@ -18,6 +18,33 @@
   <dd><%= @team.description %></dd>
 </dl>
 
+<table class="table table-striped">
+  <thead>
+    <tr>
+    </tr>
+  </thead>
+  <tbody>
+    <% @team.members.each do |member| %>
+      <tr>
+        <td>
+            <%= member.id%> <%= member.first_name%> <%= member.last_name%>
+            
+            <%= link_to t('.delete_ownership', :default => t("helpers.links.delete_ownership")),
+            delete_ownership_team_path(@team, :team_member => member.id), :method => 'post', :class => 'btn btn-default btn-xs' if @team.owners.include? member %>
+
+            <%= link_to t('.assign_ownership', :default => t("helpers.links.assign_ownership")),
+            assign_ownership_team_path(@team, :team_member => member.id), :method => 'post', :class => 'btn btn-default btn-xs' if !@team.owners.include? member  %>
+
+
+
+
+            <%= link_to t('.delete_membership', :default => t("helpers.links.delete_membership")),
```

Comment:
Maybe consider using

- the bootstrap `btn-danger` class (to make the button appear red because it deletes something)
- a confirmation dialog as in
```ruby
<%= link_to t('.destroy', :default => t("helpers.links.destroy")),
            team_path(team),
            :method => :delete,
            :data => { :confirm => t('.confirm', :default => t("helpers.links.confirm")) },
            :class => 'btn btn-xs btn-danger' %>
``` 
(taken from `teams/index.html.erb`)